package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "WebACS",
-  "version": "3.0.1",
+  "version": "3.0.0",
```

Comment:
why 3.0.0?
Is this the number of the 'first' version?
And afterwards the number will be set by the semantic versioning library automatically?